.github/workflows/run_tests.yml

```diff
@@ -112,12 +121,17 @@ jobs:
 
       - name: Build
         working-directory: ./build
-        run: cmake --build . -t ${{ matrix.target }}
+        run: |

```

Comment:
We don't need these changes, all we needed was adding blueprint, which we better try to remove as well, by removing dependency on it.